aca_preview/preview.py

```diff
@@ -391,14 +392,16 @@ def check_catalog(self):
         """
         for entry in self:
             self.check_guide_fid_position_on_ccd(entry)
+            if entry['id'] in self.guides:
```

Comment:
Right so I think I have "Yes, I would write a top-level if statement in the loop and only pass guide entries" done with this change, but this should probably be `if entry['id'] in self.guides['id']`

Reply:
Yup.

Reply:
Fixed.  Though surprised the previous code didn't error or warn.